core/src/test/java/org/infinispan/test/TestingUtil.java

```diff
@@ -222,9 +222,14 @@ public static Object extractField(Class type, Object target, String fieldName) {
             .findInterceptorExtending(interceptorToFind);
    }
 
-   public static void waitForRehashToComplete(Cache... caches) {
-      final int REHASH_TIMEOUT_SECONDS = 60; //Needs to be rather large to prevent sporadic failures on CI
-      final long giveup = System.nanoTime() + TimeUnit.SECONDS.toNanos(REHASH_TIMEOUT_SECONDS);
+   /**
+    * Waits until pendingCH() is null on all caches, currentCH.getMembers() contains all caches provided as the param
+    * and all segments have numOwners owners.
+    * @param caches
+    */
+   public static void waitForStateTransferToComplete(Cache... caches) {
```

Comment:
That's not much more clear. In #4862 I use `waitForRebalanceToComplete`. Could be `waitUntilRebalanceCompletes`, if you find that better.